### DISCUSSION TITLE
Q: Multiple resources on one page?

### DISCUSSION THREAD

**Discussion by nea** [filamentphp/filament#2947] [Help]
Ahoy

Is it possible to have multiple resources with tables and following create/edit pages from one main page?

Let's say having many configuration values that just allow a creation of a `title` with new entities used for categorization in other places. Every resource would end on their own page if I just normally generate or create the resources.

Would it be possible to "merge" them? Have the same table/create/edit logic but on the same page? e.g. in sections or something.
I saw things like having multiple forms on the same page, but not multiple tables.

Thanks a lot

**Comment by danharrin** [comment]
Yeah, you can use a custom page, and build the interface using the table builder / form builder. Very minimal HTML required. You might wanna learn a little bit of Livewire though if you don't know anything already.
